config/locales/pages/es.yml

```diff
@@ -4,3 +4,6 @@ es:
     page_not_found:
       body: Es posible que desee comprobar su enlace e intentar nuevamente. (404)
       header: La página que buscaba no existe.
+    page_took_too_long:
+      body: Es posible que desee esperar unos minutos y vuelva a intentarlo. (503)
```

Comment:
Can we change to: Por favor, es posible que desee esperar dos or tres minutos y vuelva a intentarlo.